packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -106,16 +106,16 @@ export default {
 <style lang="scss" module>
 .o-link {
     & {
-        color: $color-link-default;
+        color: $color-content-link;
     }
 
     &:hover,
     &:focus {
-        color: $color-link-hover;
+        color: darken($color-content-link, $color-hover-01);
     }
 
     &:active {
-        color: $color-link-active;
+        color: darken($color-content-link, $color-active-02);
```

Comment:
I would make this darken variable `color-active-01` to match the descriptions in the design token sheet (unless you've seen it on a figma design somewhere different to this?)